src/main.rs

```diff
@@ -12,15 +15,19 @@ use url::Url;
 mod checksum;
 use checksum::{Checksum, ChecksumValidator};
 
-static CHECKSUM_FILES: &[&str] = &[
-    "CHECKSUM.txt",
-    "checksum.txt",
-    "CHECKSUMS.txt",
-    "checksums.txt",
-    "SHASUMS256.txt",
-    "SHASUMS256",
-    // TODO add more patterns
-];
+static CHECKSUMS_FILES: Lazy<Vec<String>> = Lazy::new(|| {
+    vec![
+        "${path}/CHECKSUM.txt".to_string(),
+        "${path}/checksum.txt".to_string(),
```

Comment:
if `path` is empty, we don't get the previous value as there is an additional `/`. From my reviewer understanding it seems ok, but I just wanted to check this doesn't introduce an unexpected change in behaviour.

Reply:
I think empty path or "/" is the same.

Reply:
ok, we'll see rapidly if it causes troubles anyway